contrib/intel_env.sh

```diff
@@ -13,6 +13,8 @@ export SAMTOOLS_EXE_PATH=$SAMTOOLS_DIR/samtools
 export PICARD_PATH=$GENOMICS_DIR/ohsu/dnapipeline/picard-tools-1.110
 export GATK_JAR_PATH=/opt/gsa-unstable/target/GenomeAnalysisTK.jar
 export CP_PATH=/cluster_share/cp_pipeline
+export LABKEY_USERNAME=ccc@ccc.com
+export LABKEY_PASSWORD=Intel123
```

Comment:
This is ok for now - but ideally this should be inside a file that is an argument to the tool.